service/src/main/java/com/derpgroup/derpwizard/manager/DerpWizardManager.java

```diff
@@ -1,29 +1,55 @@
+/**
+ * Copyright (C) 2015 David Phillips
+ * Copyright (C) 2015 Eric Olson
+ * Copyright (C) 2015 Rusty Gerard
+ * Copyright (C) 2015 Paul Winters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
 package com.derpgroup.derpwizard.manager;
 
+import com.derpgroup.derpwizard.voice.model.SsmlDocumentBuilder;
+import com.derpgroup.derpwizard.voice.model.SsmlDocumentBuilder.EmphasisLevel;
 import com.derpgroup.derpwizard.voice.model.VoiceInput;
 
-public class DerpWizardManager {
+/**
+ * Manager class for dispatching input messages.
+ *
+ * @author Eric
+ * @author Rusty
+ * @since 0.0.1
+ */
+public class DerpWizardManager extends AbstractManager {
+
+  @Override
+  protected void doHelpRequest(VoiceInput voiceInput, SsmlDocumentBuilder builder) {
+    builder.endParagraph().text("I'd love to help, but I don't have any help topics programmed yet.").endParagraph();
+  }
 
-  public String handleRequest(VoiceInput vi){
-    
-    DerpWizardRequestTypes requestType = vi.getRequestName(DerpWizardRequestTypes.class);
-    
-    switch(requestType){
-    case HELLO:
-      return doHelloRequest();
-    case HELP:
-      return doHelpRequest();
-    default:
-      return "Unknown request type '" + requestType + "'.";
-    }
+  @Override
+  protected void doHelloRequest(VoiceInput voiceInput, SsmlDocumentBuilder builder) {
+    builder.endParagraph().text("Hi. This is DerpWizard.").endParagraph();
   }
 
-  private String doHelpRequest() {
-    return "<speak><p><s>I'd love to help, but I don't have any help topics programmed yet.</s></p></speak>";
+  @Override
+  protected void doGoodbyeRequest(VoiceInput voiceInput, SsmlDocumentBuilder builder) {
+    builder.endParagraph().text("Goodbye!", EmphasisLevel.MODERATE).endParagraph();
```

Comment:
For the time being, maybe.  We could just leave the method in there for future use, but have it be a no-op right now.